requirements.txt

```diff
@@ -0,0 +1,2 @@
+dockerspawner
```

Comment:
We should specify a version here too

Reply:
No, so each rebuild will fetch the latest.  A rollback means just re-use an older build.